tests/stubs/talon/__init__.py

```diff
@@ -1,12 +1,21 @@
+import inspect
```

Comment:
I'd love to get a walkthrough of the knausj testing setup at some point, as we're planning to introduce cursorless talon tests soon. If you're interested, let's find a time

Cc/ @AndreasArvidsson @will-sommers @andrewdant

Reply:
We can try and work out the timezones some time :). The main technical trick that enables all the pass shenanigans is in /conftest.py . I don't actually remember too much about that, it's just some arcane thing to hook into the class loader.

The stubs directory just has enough approximate implementation of the Talon API to allow loading of the file under test (and to facilitate testing). I don't really have any long term plans for how to extend it; I've just hacked on it when I needed something.